rwengine/src/engine/GameWorld.cpp

```diff
@@ -737,10 +737,7 @@ void GameWorld::clearCutscene() {
 bool GameWorld::isCutsceneDone() {
     if (state->currentCutscene) {
         float time = getGameTime() - state->cutsceneStartTime;
-        if (state->skipCutscene) {
-            return true;
-        }
-        return time > state->currentCutscene->tracks.duration;
+        return state->skipCutscene ? true : time > state->currentCutscene->tracks.duration;
```

Comment:
Old code was more readable.
We are not trying to win a "shortest code" competition. More explicit is probably better